sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -2,9 +2,21 @@
 
 ## 12.1.0-beta.1 (Unreleased)
 
+### Breaking Changes
```

Comment:
Minor version releases can never have breaking changes. In order to do what you are doing, you need to release version `13.0.0-beta.1`.

Also, from now on, we use new headings - https://github.com/Azure/azure-sdk-for-cpp/pull/2527#discussion_r662516141 (which we don't apply for the releases already made)

Reply:
What you can do, is to leave the old things as they were, but ignore them (most likely documenting that they have no effect).
So, the client code someone had with `12.0.0` should still compile and behave the same.

Reply:
Would need approval from Vincent for this.

Reply:
reverted